api/common/v1alpha1/tls/tls.go

```diff
@@ -102,5 +102,3 @@ var AllCiphers = []string{
 	string(EcdheRsaAes256GcmSha384),
 	string(EcdheRsaChacha20Poly1305),
 }
-
-type TlsCiphers []TlsCipher
```

Comment:
for some reason this wasn't working with `resource_generrated.zz.go` but it's just a type alias so I replaced the usages.